src/external-actor-on-team-approved-pr.ts

```diff
@@ -0,0 +1,101 @@
+import * as core from "@actions/core";
+import * as httpm from "@actions/http-client";
+import { context, getOctokit } from "@actions/github";
+
+process.on("unhandledRejection", handleError);
+main().catch(handleError);
+
+async function main(): Promise<any> {
+  const http: httpm.HttpClient = new httpm.HttpClient(
+    "external-actor-on-team-approved-pr"
```

Comment:
nit: Would be nice if the user agent is a bit more specific:
`actions/external-actor-on-team-approved-pr`